esadapter/src/main/java/com/ncc/neon/adapters/es/ElasticsearchQueryConverter.java

```diff
@@ -426,6 +429,20 @@ public static UpdateRequest convertMutationByIdQuery(MutateQuery mutateQuery) {
             .doc(mutateQuery.getFieldsWithValues());
     }
 
+    public static UpdateByQueryRequest convertMutationByFilterQuery(MutateQuery mutateQuery) {
+        SelectClause selectClause = new SelectClause(mutateQuery.getDatabaseName(), mutateQuery.getTableName());
+        QueryBuilder queryBuilder = convertWhereClauses(selectClause, List.of(mutateQuery.getWhereClause()));
+        UpdateByQueryRequest request = new UpdateByQueryRequest();
+        request.setQuery(queryBuilder);
+        request.getSearchRequest().indices("_all");
+        request.setScript(new Script(
+                ScriptType.INLINE,
+                "painless",
+                "for (entry in params.entrySet()) { ctx._source[entry.getKey()] = entry.getValue(); }",
+                mutateQuery.getFieldsWithValues()));
```

Comment:
I'd prefer to avoid ES scripts due to [security concerns](https://www.elastic.co/guide/en/elasticsearch/reference/current/modules-scripting-security.html).

Do we even need this script anyway?  Aren't the filters already set in the query builder?

Reply:
There's no `.source` method for this request type. Unfortunately this was the only way I found to specify the parameters in the request.